pkg/codegen/operations.go

```diff
@@ -211,6 +211,7 @@ func DescribeSecurityDefinition(srs openapi3.SecurityRequirements) []SecurityDef
 type OperationDefinition struct {
 	OperationID string // The operation_id description from Swagger, used to generate function names
 
+	CustomImports       []string              // Custom needed imports due to parameters being of external types
```

Comment:
Might as well rename this to imports, such that it can possibly contain other imports later on if we want to refactor that part.